scopes/dependencies/dependency-resolver/dependency-linker.ts

```diff
@@ -571,7 +572,8 @@ export class DependencyLinker {
     rootDir: string,
     name: string,
     packageName = `@teambit/${name}`,
-    skipExisting = false
+    skipExisting = false,
+    existInRootPolicy = false
```

Comment:
I think it should be
```suggestion
    existsInRootPolicy = false
```